tests/test-files.c

```diff
@@ -251,6 +251,32 @@ START_TEST(test_path_relative_child_01)
 END_TEST
 
 
+void
+test_set_absolute(const char *p, const char *expected)
+{
+    struct cork_path  *actual;
+
+    fprintf(stderr, "set_absolute(\"%s\") ?= \"%s\"\n",
+            (p == NULL)? "": p,
+            expected);
+
+    actual = cork_path_new(p);
+    cork_path_set_absolute(actual);
+    verify_path_content(actual, expected);
+    cork_path_free(actual);
+}
+
+START_TEST(test_path_set_absolute_01)
+{
+    /* We test that an absolute path really is so. */
+    DESCRIBE_TEST
+    test_set_absolute("/a", "/a");
+    test_set_absolute("/a/b", "/a/b");
+    test_set_absolute("/a/b/", "/a/b/");
```

Comment:
Tests for `"c/d"` are problematic because `cork_path_set_absolute` will prepend the `$ROOT` for the current environment.  Any thoughts on tests for that?